src/pages/chapter/forms/index.tsx

```diff
@@ -1,7 +1,108 @@
-import { Flex } from "@chakra-ui/react";
+import { AddIcon } from "@chakra-ui/icons";
```

Comment:
Once you remove chakra's icons from dependencies, you will need to replace this add icon with one from react icons.